File: Course_01/Java Language/Estrutura condicional/multiplos.java
package condicional;

import java.util.Locale;
import java.util.Scanner;

public class multiplos {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int a, b;
		
		System.out.println("Digite dois numeros inteiros:");
		a = sc.nextInt();
		b = sc.nextInt();
		
		if (a % b == 0 || b % a == 0) {
			System.out.println("São multiplos");
		}
		
		else {
			System.out.println("Não são multiplos");
		}
		sc.close();
	}

}

File: Course_01/C++ Language/EstruturaCondicional/multiplos.cpp
#include <iostream>

using namespace std;

int main()
{
    int a, b;

    cout << "Digite dois numeros inteiros: " << endl;
    cin >> a >> b;

    if (a % b == 0 || b % a == 0) {
        cout << "Sao multiplos";
    }
    else {
    cout << "Nao sao multiplos";
    }
    return 0;
}
